lib/utils.js

```diff
@@ -9,8 +9,10 @@ exports.defineCachedRelations = defineCachedRelations;
 exports.sortObjectsByIds = sortObjectsByIds;
 exports.setScopeValuesFromWhere = setScopeValuesFromWhere;
 exports.mergeQuery = mergeQuery;
+exports.createPromiseCallback = createPromiseCallback;
 
 var traverse = require('traverse');
+var Promise = require('bluebird');
```

Comment:
+1, use global `Promise` when it's available. See https://github.com/strongloop/strong-remoting/pull/179